src/main/resources/logback-spring.xml

```diff
@@ -37,6 +37,10 @@
     <appender-ref ref="consoleAppender"/>
   </logger>
 
+<!--  <logger name="org.springframework.security" additivity="false" level="DEBUG">-->
+<!--    <appender-ref ref="consoleAppender"/>-->
+<!--  </logger>-->
```

Comment:
Probably best to either remove this or set it to INFO, rather than leave commented out code.

Reply:
Makes sense. I thought this would be convenient in case we wanted to quickly enable debug logs. 
But indeed it better practice not to leave commented code. I will change this to INFO.